### ISSUE TITLE
Add enhanced possibilities to customize (accessible) name of inner components

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2396]
Scope
---
Some of PDS components have nested UI components which sometimes have predefined (accessible) names in english or no accessible name at all ("clear" button of `Textfield`). We need to ensure that all internal texts and all internal UI components can be edited from outside.

Out of Scope
---

Notes  
---
Share any further information that is relevant for processing.

Acceptance criteria
---
- [ ] All internal texts/contents must be editable from outside (either through `aria` prop or other text prop)
- [ ] "clear" button of `Textfield` component has accessible name

Subtasks
---
- [ ] Task

**Comment by stephanschroeter** [comment]
Duplicated issue: https://github.com/porsche-design-system/porsche-design-system/issues/3015
